platform/ios/CHANGELOG.md

```diff
@@ -26,7 +28,12 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Other changes
 
 * Xcode 8.0 or higher is now recommended for using this SDK. ([#8775](https://github.com/mapbox/mapbox-gl-native/pull/8775))
+<<<<<<< HEAD
```

Comment:
Whoops, I missed that one – good catch! Fixed in 1d0e203f18ba315d2f5166d671624992335d3a7c.